src/data/CassandraBackend.hpp

```diff
@@ -567,6 +567,24 @@ class BasicCassandraBackend : public BackendInterface {
         return std::nullopt;
     }
 
+    std::optional<std::uint32_t>
+    doFetchLedgerObjectSeq(ripple::uint256 const& key, std::uint32_t const sequence, boost::asio::yield_context yield)
+        const override
+    {
+        LOG(log_.debug()) << "Fetching ledger object for seq " << sequence << ", key = " << ripple::to_string(key);
+        if (auto const res = executor_.read(yield, schema_->selectObject, key, sequence); res) {
+            if (auto const result = res->template get<std::uint32_t>(); result) {
```

Comment:
I'm not sure how this works, you are trying to read a Blob as a uint32_t. Is that allowed and what was meant?

Reply:
It is true. it gets by order.

Reply:
You need to either `auto [seq, _] = get<Blob, uint32_t>` and discard the blob or add a `get<T>(std::size_t position)` and use that instead (in `data/cassandra/impl/Result.hpp`).